aws_lambda_powertools/utilities/parameters/base.py

```diff
@@ -230,6 +231,8 @@ def transform_value(value: str, transform: str, raise_on_transform_error: bool =
             return json.loads(value)
         elif transform == TRANSFORM_METHOD_BINARY:
             return base64.b64decode(value)
+        elif transform == TRANSFORM_METHOD_YAML:
+            return value  # todo
```

Comment:
__For tracking__: TODO

Reply:
@nmoutschen 
this might mean increasing the package size by adding pyyaml or soemthing of the sort. is that ok with you guys?

Reply:
Fair point - I'm wondering how frequently is YAML used and if there's any need for it. Since this is modular, I think we should keep it for another RFC unless there's a strong need to do it for AppConfig.

Size wise, PyYAML is 444KB, versus 62MB for Powertools with all dependencies (with 52MB for Botocore).